site/specs/global.yml

```diff
@@ -905,8 +945,8 @@ tags:
       Apply Configuration method.
   - name: Connectivity
     description: >-
-      Retrieve details about Bandwidth's data centres and network interconnect
-      for various services such as call termination, speech analytics, etc.
+      Retrieve details about Voxbone's data centres and network interconnect for
```

Comment:
There is plenty of reference to voxbone in the updated Global API spec. Is it after review and agreed upon or do we've separate ticket to review it?

Reply:
Yeah thats all addressed in [this](https://github.com/Bandwidth/api-specs/pull/127) PR

Additionally - any of the spec file changes are automatically pulled from [api-specs](https://github.com/Bandwidth/api-specs) - the source of truth for the PRs are in that repository 

@brianluisgomez didnt edit that `global.yml` file in his PR - it looks like its just getting pulled from main of api-specs 

running `git merge main` on this branch of api-docs should remove that `global` file change